src/System.Net.Http/tests/FunctionalTests/DiagnosticsTests.cs

```diff
@@ -381,6 +381,57 @@ public void SendAsync_ExpectedDiagnosticSourceActivityLogging()
             }, UseSocketsHttpHandler.ToString()).Dispose();
         }
 
+        [OuterLoop] // TODO: Issue #11345
```

Comment:
This is the same comment as the test below, if the issue isn't relevant to this code can it be removed? from both?